packages/now-build-utils/src/detect-builders.ts

```diff
@@ -465,6 +465,9 @@ function detectFrontBuilder(
   if (framework === 'nextjs') {
     return { src: 'package.json', use: `@vercel/next${withTag}`, config };
   }
+  if (framework === 'blitzjs') {
+    return { src: 'package.json', use: `@now/next${withTag}`, config };
```

Comment:
```suggestion
    return { src: 'package.json', use: `@vercel/next${withTag}`, config };
```

Reply:
Actually, lets change line 456 to be `if (framework === 'nextjs' || framework === 'blitzjs')`

Reply:
Fixed